packages/experimental/ActivityIndicator/src/ActivityIndicator.tsx

```diff
@@ -75,15 +79,19 @@ export const ActivityIndicator = (props: ActivityIndicatorProps) => {
     outputRange: ['0deg', '359deg'],
   });
 
-  // or display: none, but since React Native ActivityIndicator still takes up space when hidden, using opacity
-  let hideOpacity = 1;
+  // React Native ActivityIndicator still takes up space when hidden, so to perfectly match would use opacity
+  // using display: 'none' because screen reader might still read it when hidden with opacity
+  let displayValue = 'flex';
   if (props.animating == false && props.hidesWhenStopped == true) {
-    hideOpacity = 0;
+    displayValue = 'none';
   }
+
+  const activityIndicatorStyles = getActivityIndicatorStyle(40, 10);
+
   return (
-    <Animated.View style={[getActivityIndicatorStyle(40, 10).semicircle, { transform: [{rotateZ: interpolateSpin}, {perspective: 10}] }, {opacity: hideOpacity}]}>
-      <View style={getActivityIndicatorStyle(40, 10).endTop}></View>
-      <View style={getActivityIndicatorStyle(40, 10).endBottom}></View>
+    <Animated.View style={[activityIndicatorStyles.semicircle, { transform: [{rotateZ: interpolateSpin}, {perspective: 10}] }, {display: displayValue as 'flex'|'none'}]}>
```

Comment:
Is there a reason that this style is separated out into multiple objects, instead of all just being contained in `activityIndicatorStyles.semicircle`?

Reply:
They are all styling different views so I don't think they could be combined into one object. The `.semicircle` is styling the circle arc, (maybe I should change this name), the `.endTop` and `.endBottom` are styling the respective rounded ends. If they were all moved into one object, at least one of the three wouldn't be the right size or in the right spot. Is this what you meant by "contained in"?

Reply:
I meant why isn't `{ transform: [{rotateZ: interpolateSpin}, {perspective: 10}] }, {display: displayValue as 'flex'|'none'}` just part of `activityIndicatorStyles.semicircle` ?